src/Elmish.WPF.sln

```diff
@@ -68,9 +68,13 @@ Project("{6EC3EE1D-3C4E-46DD-8F32-0CC8E7565705}") = "Sticky.Core", "Samples\Stic
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Sticky", "Samples\Sticky\Sticky.csproj", "{668E3AAD-B2AA-4934-804E-234C4F502038}"
 EndProject
-Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "Capabilities.Core", "Samples\Capabilities.Core\Capabilities.Core.fsproj", "{B3BD88DC-7362-46F8-8040-6574C7B6B759}"
+Project("{6EC3EE1D-3C4E-46DD-8F32-0CC8E7565705}") = "Capabilities.Core", "Samples\Capabilities.Core\Capabilities.Core.fsproj", "{B3BD88DC-7362-46F8-8040-6574C7B6B759}"
```

Comment:
Auto-changed by my VS, I can reset if you want.

Reply:
This change is correct.  Not sure what I did to get the legacy GUID.

I never remember which one is which either.  I have to search for [this issue](https://github.com/dotnet/project-system/issues/2863) every time.